### PR TITLE
update node version and blueprint

### PR DESCRIPTION
**Pull request by rox163**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [x] #major# - incompatible API change

# CHANGELOG
**update** node version.
**update** blueprint
**remove** mirage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "ember-browserify": "~1.1.0",
     "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-code-coverage": "~0.3.0",
+    "ember-cli-code-coverage": "0.3.5",
```
Comment (dafortin) [inline package.json]: ~

**Comment by dafortin** [comment]
Approved 👍

File: package.json (hunk 1 above)
Comment (rox163) [inline package.json]: versions higher than 0.3.5 dont work. just figured that out. no root cause yet.

**Comment by dafortin** [comment]
Oh that's bad :( we did have a few repo with ~

**Comment by rox163** [comment]
yeah i only just figured it out :( atleast its just coverage and not functionality ;)

**Pull request merged**